src/packet/packet_ms.cpp

```diff
@@ -434,6 +434,27 @@ AOPacket *PacketMS::validateIcPacket(AOClient &client) const
                 client.sendServerMessage("First statement reached.");
             }
         }
+        if (l_args[4] == "=") {
+            auto l_statement = area->jumpToStatement(area->statement());
+            l_args = l_statement.first;
+            l_progress = l_statement.second;
+            client.m_pos = l_args[5];
+
+            client.sendServerMessageArea(client_name + " moved to the current statement.");
```

Comment:
This wording is a little strange.

```suggestion
            client.sendServerMessageArea(client_name + " repeated the current statement.");
```